app/components/atoms/horizontalButton/HorizontalButton.component.tsx

```diff
@@ -0,0 +1,36 @@
+import {FC} from 'react';
+
+import {
+  Arrow,
+  ButtonFace,
+  ContentText,
+  HorizontalButtonProps,
+  StyledHorizontalButton,
+} from './HorizontalButton.styles';
+
+export const HorizontalButton: FC<HorizontalButtonProps> = ({
+  content,
```

Comment:
`content` is defined as `ReactNode` but ultimately used as string. Please update this prop so its usage and scope are narrowed.